ipywidgets/static/widgets/js/widget_color.js

```diff
@@ -0,0 +1,89 @@
+
+    /*
+     * From a valid html color (named color, 6-digits or 3-digits hex format)
+     * return a 6-digits hexadecimal color #rrggbb.
+     */
+    function color2hex(color) {
+        return named_colors[color.toLowerCase()] || rgb2hex(color);
+    };
+
+    function rgb2hex(rgb) {
+        if (rgb.search("rgb") == -1) {
+            return rgb;
```

Comment:
Yeah, whatever `Color` validates, this widget should definitely support.  No more, no less.